controller/up.go

```diff
@@ -105,16 +110,16 @@ func compress(src string, buf io.Writer) error {
 		}
 
 		for _, igf := range ignoreFiles {
-			if strings.HasPrefix(file, igf.prefix) { // if ignore file applicable
-				trimmed := strings.TrimPrefix(file, igf.prefix)
+			if strings.HasPrefix(relativeFile, igf.prefix) { // if ignore file applicable
+				trimmed := strings.TrimPrefix(relativeFile, igf.prefix)
```

Comment:
Does this still work with relative file paths instead of absolute?

Reply:
The previous paths were all relative to the current directory so this is just matching that behaviour. The ignore paths are relative (e.g. node_modules) so we need to compare against the path relative to the project root.